### ISSUE TITLE
Ugly wrapped generics

### ISSUE THREAD

**Issue by victornoel** [thorbenvh8/prettier-java#39]
I got the following (inside a class)
```java
  private final Map<Integer,
  DbCategorisationLevel> levels = new HashMap<>();
```

While I would have expected:
```java
  private final Map<Integer, DbCategorisationLevel> levels = new HashMap<>();
```

**Comment by thorbenvh8** [comment]
maybe give this a shot yourself? :)

**Comment by victornoel** [comment]
This was closed by 88145f8
